setup.py

```diff
@@ -20,6 +20,7 @@
     install_requires=[
         "boto3>=1.5.8",
         "dataclasses;python_version<'3.7'",
+        "flake8<=4.0.1"
```

Comment:
let's not pin this here. It's not a runtime dependency and will prevent all downstream dependants from upgrading at their own pace.  Once this branch is updated from master this should appear in the "lint" extra instead.

And I guess we hope the maintainers of [flake8-logging-format](https://github.com/globality-corp/flake8-logging-format) fix their builds 😳?